pkgs/by-name/mi/misskey/package.nix

```diff
@@ -0,0 +1,124 @@
+{ stdenv
+, lib
+, nixosTests
+, fetchFromGitHub
+, nodejs
+, pnpm
+, makeWrapper
+, python3
+, bash
+, jemalloc
+, ffmpeg-headless
+, writeShellScript
+, ...
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "misskey";
+
+  version = "2024.5.0";
+
+  src = fetchFromGitHub {
+    owner = "misskey-dev";
+    repo = finalAttrs.pname;
+    rev = finalAttrs.version;
+    hash = "sha256-nKf+SfuF6MQtNO53E6vN9CMDvQzKMv3PrD6gs9Qa86w=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    nodejs
+    pnpm.configHook
+    makeWrapper
+    python3
+  ];
+
+  patches = [
+    ./pnpm_version.patch
```

Comment:
Could you add a comment here explaining the reason for the patch and if there is a point in time / a condition when it should be dropped ?

Reply:
..appearently that patch is not necessary after all, I thought I got build failures without it in the beginning but the package seems to build fine without it, so I removed it again.